src/api/search/src/routes/query.js

```diff
@@ -12,4 +12,13 @@ router.get('/', validateQuery, async (req, res, next) => {
   }
 });
 
+// query for autocomplete
+router.get('/complete', validateQuery, async (req, res, next) => {
```

Comment:
I wonder if we should do `/authors/autocomplete` so that we can do `/posts/autocomplete` later etc.?

Reply:
Sure, I can change it to that, cause its more straightforward.
I also think its also possible to do `multi_match` queries on multiple autocomplete fields in the future too, and we can always modify what is returned from the queries. This is just assumption based off of my research, nothing concrete yet.